frontend/src/components/NewRunParametersV2.test.tsx

```diff
@@ -75,9 +86,24 @@ describe('NewRunParametersV2', () => {
     fireEvent.change(strParam, { target: { value: 'new string' } });
     expect(strParam.closest('input').value).toEqual('new string');
 
-    const intParam = screen.getByDisplayValue(123);
+    const intParam = screen.getByDisplayValue('123');
     fireEvent.change(intParam, { target: { value: 999 } });
     expect(intParam.closest('input').value).toEqual('999');
+
+    const boolParam = screen.getByDisplayValue('true');
+    fireEvent.change(boolParam, { target: { value: false } });
```

Comment:
Shouldn't it be `False` instead of `false`?

Reply:
This error is eliminated after change the design of the `convertInput` function.